registry.json

```diff
@@ -3472,6 +3472,21 @@
     "operatorName": "Felix",
     "github": "lix-pham"
   },
+  {
+    "ticker": "GOLGI",
+    "poolId": "effc78f36adf09718712f78a4ae84f02e6d7cd79e3c9492fcaa2a7f5",
+    "social": {
+      "facebook": "Golgi-Stake-Pool-100691725510828",
+      "twitter": "GolgiIt",
+      "website": "https://golgi-pool.it",
+      "reddit": "",
+      "instagram": "golgistakepool",
+      "youtube": ""
+    },
+    "telegram": "[GOLGI] Stake Pool channel",
```

Comment:
Can you please put your personal telegram handle here?

Reply:
yes thanks.
I have update with my telegram handle: @Giampiero_GOLGI